payments-core/src/androidTest/java/com/stripe/android/view/CardInputWidgetTest.kt

```diff
@@ -0,0 +1,103 @@
+package com.stripe.android.view
+
+import androidx.activity.ComponentActivity
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveableStateHolder
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.viewinterop.AndroidView
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.typeText
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withHint
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import com.stripe.android.PaymentConfiguration
+import com.stripe.android.R
+import org.junit.Rule
+import org.junit.Test
+
+class CardInputWidgetTest {
```

Comment:
Right now this test isn't being run. Wondering if I should add another instrumentation task to the Bitrise config or move this test to the example instrumentation tests.